src/components/file-upload/_file-upload.scss

```diff
@@ -7,13 +7,66 @@
 @import "../label/label";
 
 @include govuk-exports("govuk/component/file-upload") {
+  $spacing-unit: govuk-spacing(1);
```

Comment:
Can we think of a more descriptive name for this?

Reply:
Updated!